packages/expandable-search-bar/src/expandable-search-bar.ts

```diff
@@ -0,0 +1,261 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  CSSResult,
+  TemplateResult,
+} from 'lit-element';
+
+import magnifyingGlassIcon from './assets/img/magnifying-glass';
+import disclosureTriangleIcon from './assets/img/disclosure';
+import clearResultsIcon from './assets/img/clear-results';
+
+import QuickSearchEntry from './models/quick-search-entry';
+
+import './quick-search';
+
+@customElement('expandable-search-bar')
+export default class ExpandableSearchBar extends LitElement {
+  @property({ type: Boolean }) isOpen = false;
+
+  @property({ type: Boolean }) showsDisclosure = false;
+
+  @property({ type: String }) searchTerm = '';
+
+  @property({ type: Array }) quickSearches: QuickSearchEntry[] = [];
+
+  render(): TemplateResult {
+    return html`
+      <div
+        class="
+          container
+          ${this.isOpen ? 'is-open' : ''}
+          ${this.showsDisclosure ? 'shows-disclosure' : ''}"
+      >
+        <div class="search-bar ${this.searchTerm === '' ? '' : 'is-searching'}">
+          <div class="magnifier-container endcap">
```

Comment:
can we wrap the `svg` in an image tag to give it proper `alt` text?

Reply:
I'm writing out the full `<svg>` tag inline here so I can't wrap that in an `<img>` tag _but_ I learned today that you can add a `<title>` tag to the `<svg>` and it will be used as title text and screen readers will use it. Some more deets on svg accessibility: https://css-tricks.com/accessible-svgs/#article-header-id-1

![Screen Shot 2019-11-12 at 11 03 46 AM](https://user-images.githubusercontent.com/51138/68702321-773f4300-053d-11ea-8e4b-446820252fb2.png)